Src/Coravel/Coravel.csproj

```diff
@@ -14,9 +14,10 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.1.0"/>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.0"/>
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="2.1.0"/>
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.1.0" />
```

Comment:
Adding this dep might not be a big deal (for me and you), but there has already been comments around limiting dependencies in Coravel.  I need to consider how others are using Coravel - which does include Micro-service situations. 

So this isn't "bad", but if there's a way to avoid adding another dependency then at least on paper it's "cleaner".

See my next comment for a potential alternative.